frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -1,20 +1,25 @@
-import { ClickAwayListener, PopperChildrenProps } from '@mui/base';
-import { Grow, Paper, Popper } from '@mui/material';
-import { Dispatch, ReactElement, RefObject, SetStateAction } from 'react';
+import { ClickAwayListener, Portal } from '@mui/base';
```

Comment:
sorry another comment that I just noticed from some other work I was doing - shouldn't we import `ClickAwayListener` and `Poral` from `@mui/material`?

Reply:
It seems to work fine. We use both throughout our repo. Maybe a TD item?